types/knockout/index.d.ts

```diff
@@ -4,21 +4,23 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 
-interface KnockoutSubscribableFunctions<T> {
+interface KnockoutExtensionFunctions {
     [key: string]: KnockoutBindingHandler | undefined;
+}
 
-	notifySubscribers(valueToWrite?: T, event?: string): void;
+interface KnockoutSubscribableFunctions<T> {
+    notifySubscribers(valueToWrite?: T, event?: string): void;
 }
+type KnockoutSubscribableExtensionFunctions<T> = KnockoutSubscribableFunctions<T> & KnockoutExtensionFunctions;
```

Comment:
That is bad, because it means that it really does not compliant `KnockoutExtensionFunctions` interface. Looks like TypeScript just doesn't catch such errors yet, but will do in the future. `&` basically says that values can be both of these types at the same time. Which is not the case for deferred updates.
To solve this need to rethink what is the extension functions. Do they really need to be `KnockoutBindingHandler`? Maybe just remove this requirement?